neurons/allocate.py

```diff
@@ -48,7 +48,6 @@ def get_config() -> bt.config:
     parser = argparse.ArgumentParser(description="Configure the database generation.")
     parser.add_argument("--db_root_path", default="~/tensorage-db", help="Path to the data database.")
     parser.add_argument("--size_in_gb", type=float, default=MIN_SIZE_IN_GB, help="Size of path to fill.")
-    parser.add_argument("--validator", action="store_true", default=False, help="If True, only generate hash DB for validators.")
```

Comment:
I think, we need this argument to specify whether it's validator or miner.
If validator, generate hashes only, but if miner, data + hash.